course_catalog/constants.py

```diff
@@ -35,9 +35,19 @@ class AvailabilityType(Enum):
     archived = "Archived"
 
 
+class ListType(Enum):
+    """
+    Enum for UserLists. User-created lists that are not Learning Paths are displayed
+    to the user as "Your Lists"
+    """
+
+    learning_path = "Learning Path"
+    list = "List"
```

Comment:
Change to not use a python builtin as a constant. Uppercasing the constants would fix that.

Reply:
👍